### ISSUE TITLE
[Question]  Loading file in steps

### ISSUE THREAD

**Issue by mogui** [citrusframework/yaks#387]
Hi all,maybe it's just me missing something.

I'm trying to load the request body for an http send step: `Given load HTTP request body user.yaml`, but it fails everytime saying `Failed to load body from file resource user.yaml`.
The file is in my local machine in teh same directory of the feature file.
Am I miss something? 

I assume the file is somehow loaded in the container as a volume, isn't it? If not how can it work?

**Comment by christophd** [comment]
yes you have to add the resource to the test container. You can use the `yaks-config.yaml` to add resources to the runtime.

something like:
```yaml
config:
  runtime:
    resources:
      - foo.yaml
```

**Comment by christophd** [comment]
if `yaks-config.yaml` is in the very same directory as the feature file it is automatically loaded
